src/Handlers/AssetsEnqueueHandler.php

```diff
@@ -95,6 +95,8 @@ protected function _enqueueAssets($assetsUrlMap, $assets)
             ),
         ]);
 
+        wp_localize_script('eddbk-wizard-app-require', 'EDDBK_WIZARD_REQUIRE_BASE_URL', $assets->get('scripts_base_path'));
```

Comment:
I'm pretty sure that the third parameter must be an array. So, you cannot just send 1 variable with 1 value; you have to send a map of values.

Reply:
No, it shouldn't be array. I tested this solution in the browser, and it works as expected.

Reply:
[Codex](https://codex.wordpress.org/Function_Reference/wp_localize_script) says it should. Besides, are you going to add `wp_localize_script()` for every value you might want to pass? Better go with the spec.